packages/googlecloud/functions/getanswer/inquirer.py

```diff
@@ -218,7 +218,7 @@ def route_question(db_general, db_in_depth, db_voting, query, query_type, k=4):
     elif query_type == RESPONSE_TYPE_GENERAL:
         return get_general_summary_response_from_query(db_general, query, k)
     elif query_type == RESPONSE_TYPE_VOTING:
-        return get_voting_summary_response_from_query(db_voting, query, k=4)
+        return get_voting_summary_response_from_query(db_voting, query, k=1)
```

Comment:
@ayyubibrahimi heads up that some ordinances go through committee. In which case they will have a vote there too as well.